easybuild/easyconfigs/f/FFTW/FFTW_3.3.4-p2.patch

```diff
@@ -0,0 +1,871 @@
+#https://www-user.tu-chemnitz.de/~potts/workgroup/pippig/software.php.en#scripts
+#Within these scripts we apply the following patches that have been submitted to FFTW but are not yet included in FFTW-3.3.4:
+#We add two more global transposition algorithms to the planner.
+#We patch file mpi/transpose-pairwise.c in order to fix a double free bug.
```

Comment:
Yes, ok, but you got it from the script now, right?

I would make it clear in the patch name that it's related to PFFT.

Maybe `FFTW-3.3.4_PFFT-20160307.patch`, or with a more suitable datestamp (i.e. when the script was last updated).